### PR TITLE
Backports 0.19 guix

### PR DESCRIPTION
**Pull request by Munkybooty**

### CODE REVIEW

File: depends/packages/qt.mk [hunk 1]
```diff
@@ -215,7 +215,8 @@ define $(package)_preprocess_cmds
   sed -i.old "0,/^QMAKE_LFLAGS_/s|^QMAKE_LFLAGS_|!host_build: QMAKE_LFLAGS            = $($(package)_ldflags)\n&|" qtbase/mkspecs/win32-g++/qmake.conf && \
   sed -i.old "s|QMAKE_CC                = clang|QMAKE_CC                = $($(package)_cc)|" qtbase/mkspecs/common/clang.conf && \
   sed -i.old "s|QMAKE_CXX               = clang++|QMAKE_CXX               = $($(package)_cxx)|" qtbase/mkspecs/common/clang.conf && \
-  sed -i.old "s/error(\"failed to parse default search paths from compiler output\")/\!darwin: error(\"failed to parse default search paths from compiler output\")/g" qtbase/mkspecs/features/toolchain.prf
+  sed -i.old "s/error(\"failed to parse default search paths from compiler output\")/\!darwin: error(\"failed to parse default search paths from compiler output\")/g" qtbase/mkspecs/features/toolchain.prf \
```
Comment (UdjinM6) [inline depends/packages/qt.mk:218]: 15277:
```suggestion
  sed -i.old "s/error(\"failed to parse default search paths from compiler output\")/\!darwin: error(\"failed to parse default search paths from compiler output\")/g" qtbase/mkspecs/features/toolchain.prf && \
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Review by UdjinM6 — APPROVED** [review]
utACK assuming we are going to dashify this later

**Comment by PastaPastaPasta** [comment]
@Munkybooty do future (non-guix) backports rely on this? Or can we delay this until v19?

**Comment by Munkybooty** [comment]
I think we should be safe delaying this, all the files are basically isolated and guix seems to be untouched until v20 backports. The only merge conflicts I can see appearing are with qt.mk other than that, it should be fine

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Comment by knst** [comment]
superseeded by:
 - https://github.com/dashpay/dash/pull/5194 
 - https://github.com/dashpay/dash/pull/5237
 - https://github.com/dashpay/dash/pull/5285

**Pull request closed without merging**